db.go

```diff
@@ -55,6 +55,12 @@ type DB struct {
 	// THIS IS UNSAFE. PLEASE USE WITH CAUTION.
 	NoSync bool
 
+	// When true, skips the truncate call when resizing the database.
+	// Setting this to true is only safe on non-ext3/ext4 systems.
+	//
```

Comment:
Would it be worth adding a note here the we found that Truncate can be very slow on some filesystems?